packages/framework-provider-azure-infrastructure/src/infrastructure/helper/function-zip.ts

```diff
@@ -0,0 +1,117 @@
+import { BoosterConfig } from '@boostercloud/framework-types'
+import { GraphqlFunction } from '../functions/graphql-function'
+import { EventHandlerFunction } from '../functions/event-handler-function'
+import { SchedulesFunctions } from '../functions/schedules-functions'
+import { FunctionDefinition } from '../types/functionDefinition'
+import archiver = require('archiver')
```

Comment:
Why the `= require(...)`? Why not importing directly?

Reply:
Ops! some wrong auto import. let me fix it